build/build_conda.sh

```
@@ -67,54 +74,39 @@ download_pipeline() {
-    hydra-genetics prepare-environment container-path-update -c ${PIPELINE_NAME}_${TAG_OR_BRANCH}/${PIPELINE_NAME}/config/config.yaml -n config.new.yaml -p $APPTAINER_CACHE
-    mv config.new.yaml ${PIPELINE_NAME}_${TAG_OR_BRANCH}/${PIPELINE_NAME}/config/config.yaml 
+    hydra-genetics prepare-environment create-singularity-files \
+        -c "${pipeline_dir}/${PIPELINE_NAME}/config/config.yaml" \
+        -o apptainer_cache \
+        || { echo "ERROR: Failed to create singularity files"; exit 1; }
+
+    # Update the paths to the containers in the config before archiving
+    hydra-genetics prepare-environment container-path-update \
+        -c "${pipeline_dir}/${PIPELINE_NAME}/config/config.yaml" \
+        -n "config.new.yaml" \
+        -p $APPTAINER_CACHE \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)